### ISSUE TITLE
Parameterized query not working with CREATE TABLE

### ISSUE THREAD

**Issue by alaindeseine** [tekartik/sqflite#846]
When trying to create a new table either with `rawQuery` or `execute` methods with table name as a parameter, it return an error.

SQFLite version: 2.0.2+1

Source code to reproduce: 

````
String name = 'myTable';
await db.rawQuery("CREATE TABLE ? ( id INTEGER NOT NULL, name TEXT)", [name]);
````

Error:
````
I/flutter (22910): error DatabaseException(near "?": syntax error (Sqlite code 1): , while compiling: create table ? ( id INTEGER NOT NULL, name TEXT), (OS error - 2:No such file or directory)) sql 'create table ? ( id INTEGER NOT NULL, name TEXT)' args [myTable] during open, closing...
E/flutter (22910): [ERROR:flutter/lib/ui/ui_dart_state.cc(198)] Unhandled Exception: DatabaseException(near "?": syntax error (Sqlite code 1): , while compiling: create table ? ( id INTEGER NOT NULL, name TEXT), (OS error - 2:No such file or directory)) sql 'create table ? ( id INTEGER NOT NULL, name TEXT)' args [myTable]
````

It seems to me like it try to open the OS file before replace parameters.

**Comment by alextekartik** [comment]
I don't think you can put the table name as a parameter. I think parameters are only for values (in insert, update and where clause). Instead do `CREATE TABLE myTable...`

Also query are only for `SELECT` statement, you should use `db.execute` instead

**Comment by alaindeseine** [comment]
There is no problem execute CREATE statement with ``rawQuery``, it works well!

For the impossibility of parameterize table name in CREATE statement it's not documented. As `execute` (or `rawQuery`) authorize passing an array of parameters, i thinks that if it can't be used for CREATE statement it should be documented or returned error should be more explicit.

**Comment by alextekartik** [comment]
Yes rawQuery works ok for CREATE statement but it does not have any returned value so prefer execute here.

sqflite does not do any parsing and assume some level of knowledge of SQLite. The Android API (which is similar) does not mention that neither. The error returned is whatever the native layer returns (so it could depend on the platform). Sometimes the error is not very helpful unfortunately.

Sorry for the poor documentation and for learning the hard way. PR on the doc is welcome if you find a good place to mention that...
